src/sentry/utils/email.py

```diff
@@ -336,7 +336,7 @@ def build(self, to, reply_to=None, cc=None, bcc=None):
                 headers.setdefault('References', thread.msgid)
 
         msg = EmailMultiAlternatives(
-            subject=subject,
+            subject=subject.splitlines()[0],
```

Comment:
Choosing to use `splitlines` over `split` since we see this as `\r\n` as well, so we need to split all cases.

Also worth noting that this will error for an empty string, but I'm pretty confident we should never attempt to send an email with an empty subject, so should be fine. I'd be ok adding some guard here if we think that's possible, or an assertion elsewhere to make sure that's not true.